met-web/src/components/comments/admin/reviewListing/index.tsx

```diff
@@ -195,7 +195,7 @@ const SubmissionListing = () => {
                 <MetTable
                     headCells={headCells}
                     rows={submissions}
-                    noRowBorder={true}
+                    noRowBorder={false}
```

Comment:
is noRowBorder = false the default? then we can remove it rather than changing to false?

Not sure whats better..just checking

Reply:
I think leaving it might be better so future devs know this is an option